iOS/Flows/New Channel/NewChannelCollectionViewManager.swift

```diff
@@ -29,19 +29,24 @@ class NewChannelCollectionViewManger: CollectionViewManager<NewChannelCollection
         self.allowMultipleSelection = true 
 
         self.collectionView.animationView.play()
-        GetAllConnections().makeSynchronousRequest(andUpdate: [], viewsToIgnore: [])
-            .mainSink { result in
-                switch result {
-                case .success(let connections):
-                    self.connections = connections.filter { (connection) -> Bool in
-                        return !connection.nonMeUser.isNil
-                    }
-                    self.loadSnapshot()
-                case .error(_):
-                    break
-                }
-                self.collectionView.animationView.stop()
-            }.store(in: &self.cancellables)
+
+        Task {
+            await self.getAllConnections()
+        }
+    }
+
+    private func getAllConnections() async {
+        do {
+            let connections = try await GetAllConnections().makeRequest(andUpdate: [], viewsToIgnore: [])
+            self.connections = connections.filter { (connection) -> Bool in
+                return !connection.nonMeUser.isNil
+            }
+            self.loadSnapshot()
+        } catch {
+            print(error)
+        }
+
+        self.collectionView.animationView.stop()
```

Comment:
To stop the animation for view for both success and failure, I put this outside the do catch.

Reply:
Ideally the animation view is handled by statusable, and passed in to the makeRequest call.